### PR TITLE
A bit of R# cleanup with StyleCop rules

### PR DESCRIPTION
**Pull request by wforney**

### CODE REVIEW

**Comment by wforney** [comment]
Well, the stuff it generates is a bit redundant at times, but if you're building a bigger application it can come in handy as a starting point for writing the docs, assuming you're going to bother with docs. This little project may not require them but getting into the habit of following the patterns is a good thing most of the time.

Yes, R# is a paid VS add on, but there are a lot of other free ones that generate this stuff too. I just happen to use R# in my own work so it's handy. I'm not trying to force my style on anybody, I just put it out there and if they want to use it then fine. If they don't, that's fine too. :)

**Comment by luisrudge** [comment]
Sorry if sounded like that, but I didn't mean to imply that you were forcing anything. Anyway, I'm not a maintainer, I was just sharing my 2 cents

File: src/live.asp.net/AppSettings.cs [hunk 1]
```diff
@@ -1,20 +1,67 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved. 
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+﻿// --------------------------------------------------------------------------------------------------------------------
```
Comment (Eilon) [inline src/live.asp.net/AppSettings.cs:1]: Please do not change the copyright and license headers.

File: src/live.asp.net/Controllers/AccountController.cs [hunk 2]
```diff
@@ -1,45 +1,77 @@
...
+//   Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+// </copyright>
+// <summary>
+//   The account controller.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
 
 namespace live.asp.net.Controllers
 {
+    using System.Threading.Tasks;
```
Comment (Eilon) [inline src/live.asp.net/Controllers/AccountController.cs:13]: Please keep namespaces formatted as they were.

**Comment by Eilon** [comment]
I'm not sure I see a lot of value in this change. It's great to add meaningful doc comments where they weren't present, but doc comments that just repeat the member name seem like more noise than value. Also, changing the formatting of files is not necessary.

**Comment by DamianEdwards** [comment]
Concur with @Eilon in this case.

**Comment by wforney** [comment]
When learning a codebase I go through every file and run ReSharper cleanup on it. It's just my way of getting familiar with the code. As I mentioned in response to some comments, I don't particularly care if you use the pull request code or not, or whether you approve of the formatting or not.

It wasn't sent with a thought to its use in the way that has been mentioned and I'd have not bothered to send the PR if I'd known it would get me openly mocked on Twitter.

ProTip: Don't mock people just because they don't agree with your formatting or because they actually follow FxCop/StyleCop rules.

And I do agree that the comment formatting and regions may not add a lot of value in this case, so don't get me wrong I didn't expect you guys to use this as is. That's just a bit of a misconception about PRs on my part, so I've cleared that up now.

**Comment by shanselman** [comment]
Hm? My tweet had nothing to do with you. I'm working on firsttimersonly.com and LOTS of folks who aren't you tend to run Resharper as a part of some PR. It's a very common "meme" or pattern. 

I'm sorry that you felt my silly tweet was directed at you. I wasn't aware of this PR and didn't intend to target or offend you.

**Pull request closed without merging**